### ISSUE TITLE
Deeper Exercise: Not Detecting New Classes

### ISSUE THREAD

**Issue by henrylao** [akailany/Pattern-Segmentation-in-Aerial-Agricultural-Images#8]
**Detector Not Detecting New Dataset Classes**
The default dataset for `yolov5s.pt` consists of __80 classes__ where __traffic light__ and __stop sign__ from the [New 
Road Sign Detection Dataset](https://www.kaggle.com/andrewmvd/road-sign-detection). Despite training with forecasting using a validation set, why does the model NOT detect __crosswalk__ and __speed limit__ when deploying to Android?
 - Option 1: Use the pretrained weights and train with more epochs to account for the new classes to hopefully overcome pretrained model's priors
 - Option 2: Train from scratch following: https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data using `--weights '' --cfg yolov5s.yaml` where weights are randomly initialized
- __UPDATED__ per (11/09) meeting with Professor Grewe -- remove the top layer so that we can reconfigure how the model activates and forecasts. __NOTE__ since we're using PyTorch, the removal of the top-layer and reconfiguration will require modifications in the `yaml` file that defines a model's architecture @manishkakarla @PradeepG00 @akailany 
           - Related Issue from YoloV5: https://github.com/ultralytics/yolov5/issues/1418
           - File to Modify if we plan to use `small yolo v5`: https://github.com/ultralytics/yolov5/blob/199c9c787427ece5723d5309e1c7c524a99bc59d/models/yolov5s.yaml#L47
